lab_1/main.py

```diff
@@ -51,7 +81,12 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
+    freq_dict = calculate_frequencies(remove_stop_words(tokenize(text), stop_words))
+    if type(language) == str and type(text) == str and type(stop_words) == list:
```

Comment:
There should be a check for the input arguments at the top of the function - after the documentation string. Remove nesting here and extract the check into a separate `if` block earlier.